integration_tests/steps/integration_tests.py

```diff
@@ -660,9 +661,10 @@ def run_sim(context, scenario, total_duration, slot_length, tick_length, market_
     paused = False
     pause_after = duration()
     repl = False
-    no_export = True
-    export_path = None
+    no_export = False
```

Comment:
Hmm, this will delay the integrationtests that dont need exported results (and adds up significantly to the runtime of all tests). I suggest to also parse this as an step argument.

Reply:
I would rather add it in the context actually, and use an extra 'given' step in order to set it. That way we will avoid to change lots of step definitions in the feature files.